resources/scss/_builder.scss

```diff
@@ -227,6 +227,9 @@
     .hundredPercent {
         width: 100%;
     }
+    .fiftyPercent {
+        width: 50%;
+    }
```

Comment:
Tailwind has a native class for this, `w-1/2`.

https://tailwindcss.com/docs/width

Reply:
Fixed - good catch, thanks